modules/apps/asset/asset-list-test/src/testIntegration/java/com/liferay/asset/list/service/test/AssetListEntrySegmentsEntryRelServiceTest.java

```diff
@@ -211,6 +242,61 @@ public void testUpdateAssetListEntrySegmentsEntryRelTypeSettings()
 			typeSettingsUpdated);
 	}
 
+	@Test
+	public void testUpdateVariationsPriority() throws Exception {
+		AssetListEntry assetListEntry = AssetListTestUtil.addAssetListEntry(
+			_group.getGroupId());
+
+		AssetListEntrySegmentsEntryRel assetListEntrySegmentsEntryRel1 =
+			AssetListTestUtil.addAssetListEntrySegmentsEntryRel(
+				_group.getGroupId(), assetListEntry);
+		AssetListEntrySegmentsEntryRel assetListEntrySegmentsEntryRel2 =
+			AssetListTestUtil.addAssetListEntrySegmentsEntryRel(
+				_group.getGroupId(), assetListEntry);
+		AssetListEntrySegmentsEntryRel assetListEntrySegmentsEntryRel3 =
+			AssetListTestUtil.addAssetListEntrySegmentsEntryRel(
+				_group.getGroupId(), assetListEntry);
+
+		long[] priorities = {
+			assetListEntrySegmentsEntryRel3.
+				getAssetListEntrySegmentsEntryRelId(),
+			assetListEntrySegmentsEntryRel1.
+				getAssetListEntrySegmentsEntryRelId(),
+			assetListEntrySegmentsEntryRel2.
+				getAssetListEntrySegmentsEntryRelId()
+		};
+
+		_assetListEntrySegmentsEntryRelLocalService.updateVariationsPriority(
+			priorities);
+
+		AssetListEntrySegmentsEntryRel assetListEntrySegmentsEntryRelUpdated1 =
```

Comment:
The variable name must end with the type, so instead of `assetListEntrySegmentsEntryRelUpdated1` use `updatedAassetListEntrySegmentsEntryRel1`.

Reply:
All 3 variables updated to updatedAssetListEntrySegmentsEntryRelX